### DISCUSSION TITLE
Fix typos in web documentation ?

### DISCUSSION THREAD

**Discussion by WeisLeDocto** [thelfer/tfel#613] [General]
Hi there,

I started reading the online documentation, and noticed quite many typos here and there already.
I'll be reading the docs quite extensively in the coming weeks, so I'll likely spot more of them.
Would a PR correcting them be welcome here ? I know it's probably super low priority.

**Comment by thelfer** [comment]
Hi @WeisLeDocto, that would be of great help !

> I know it's probably super low priority.

No it's not, not at all. It's just that the main writter of the documentation (myself) is extremly bad at reading its own text...

**Reply by WeisLeDocto** [reply]
Perfect, I'll submit a PR when I'll be further in the docs !

**Comment by thelfer** [comment]
By the way, we started to write the MFront book. Could be useful for a beginner. Let me know if you wish a draft

**Reply by WeisLeDocto** [reply]
Sounds great, I'm interested indeed

**Reply by thelfer** [reply]
send me an e-email: thomas dot helfer at cea dot fr
